frontend/interfaces/config.ts

```diff
@@ -11,6 +11,7 @@ import { IIntegrations } from "./integration";
 export default PropTypes.shape({
   org_name: PropTypes.string,
   org_logo_url: PropTypes.string,
+  contact_url: PropTypes.string,
```

Comment:
Added several missing keys to config interface. HMW ensure our frontend interface is updated when the backend adds new keys to an API?